script/cibuild

```diff
@@ -27,4 +28,14 @@ echo "## tests/coverage ########################################################
 script/coverage
 echo "## validate setup.py build #####################################################"
 python setup.py build
+echo "## validate setup.py install ###################################################"
+deactivate
+TMP_DIR=$(mktemp -d -t ci-XXXXXXXXXX)
+python3 -m venv $TMP_DIR
+. "$TMP_DIR/bin/activate"
+python setup.py install
+octodns-sync --help
+echo "## validate tests can run against installed code ###############################"
+pip install pytest pytest-network
+pytest
```

Comment:
Did you want this to be `pytest --no-network` per the PR description?

Also looks like we're running coverage test as well as a separate pytest. Any way we could run the full coverage test suite just once on the installed code to speed things up? May be even put the install+test approach in `script/coverage` itself? That way the pre commit hook will also work more similarly as the CI build.

Reply:
Yup. I originally was trying other things to install the test requirements and had to remove the no-network temporarily and forgot to put it back. Will do.